### PR TITLE
Use ABIs and bins from @openstfoundation/mosaic-contracts

### PR DESCRIPTION
**Pull request by hobofan**
Depends on #34.

Currently blocked, because the `OSTPrime` contract is not being exposed from the package. Otherwise, it seems that the ABIs were already at the latest status, and this doesn't seem to break anything.

### CODE REVIEW

File: test_integration/01_chain_setup/00_config.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+describe('Setup Helpers - Config', () => {
+  // doing this in a test instead of before hook,
+  // because before hook doesn't trigger when there are no tests
+  it('should set setupConfig', async () => {
+    const accountsOrigin = await shared.origin.web3.eth.getAccounts();
+    shared.setupConfig = {
+      chainOwner: accountsOrigin[0],
+      deployerAddress: accountsOrigin[0],
+      organizationOwner: accountsOrigin[1],
```
Comment (schemar) [inline test_integration/01_chain_setup/00_config.js]: Isn't the deployer the organization owner?

File: test_integration/01_chain_setup/01_chain_setup.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+const { assert } = require('chai');
+const Web3 = require('web3');
+
+const shared = require('../shared');
+const { ChainSetup } = require('../../index');
+
+const MockContractsDeployer = require('../../tests/utils/MockContractsDeployer');
+
+describe('ChainSetup', () => {
+  let caMockToken;
```
Comment (schemar) [inline test_integration/01_chain_setup/01_chain_setup.js]: What is `ca`? 🤔

File: test_integration/01_chain_setup/01_chain_setup.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+      libs: {
+        deployer: config.deployerAddress
+      },
+      cogateway: {
+        deployer: config.deployerAddress,
+        bounty: '100'
+      }
+    };
+
+    return helper.setup(valueToken, originConfig, auxiliaryConfig).then((output) => {
```
Comment (schemar) [inline test_integration/01_chain_setup/01_chain_setup.js]: Could it be `setup` instead of `helper.setup`?

**Review by schemar — CHANGES_REQUESTED** [review]
Nice ⛵️ 
I added my questions.

Also:
* Is there a ticket/plan to get rid of the webpack stuff?
* Why are there a bunch of `..._helper.js`? Should they instead be called `..._deployer.js` or similar?

File: test_integration/01_chain_setup/00_config.js (hunk 1 above)
Comment (hobofan) [reply test_integration/01_chain_setup/00_config.js]: Doesn't have to be. `owner` can be supplied as a param the Organization constructor.

File: test_integration/01_chain_setup/01_chain_setup.js (hunk 2 above)
Comment (hobofan) [reply test_integration/01_chain_setup/01_chain_setup.js]: `ca` is contract address. I've usually cleaned it up to `address`, but looks like I missed this one.

**Comment by hobofan** [comment]
@schemar Can we please do the review and related discussion in #34? This PR is supposed to be only about the most recent commit.

**Comment by hobofan** [comment]
Updated to `alpha.1` version which contains the fix from https://github.com/OpenSTFoundation/mosaic-contracts/pull/637, and everything seems to still work.

File: libs/AbiBinProvider.js [hunk 4]
```diff
@@ -101,13 +103,7 @@ class AbiBinProvider {
...
     }
 
-    //__NOT_FOR_WEB__BEGIN__
-    let fPath = path.resolve(oThis.binFolderPath, contractName + '.bin');
-    let bin = fs.readFileSync(fPath, 'utf8');
-    if (typeof bin === 'string' && bin.indexOf('0x') != 0) {
-      bin = '0x' + bin;
-    }
-    //__NOT_FOR_WEB__END__
+    const bin = mosaicContracts[contractName].bin;
```
Comment (schemar) [inline libs/AbiBinProvider.js]: Please note that .bin is optional, e.g. for interfaces.

**Review by schemar — APPROVED** [review]
Nice!! 🎉

Just note that .bin may not exist in a contract.

File: libs/AbiBinProvider.js (hunk 4 above)
Comment (hobofan) [reply libs/AbiBinProvider.js]: Good point! I'll add a check for it.

**Pull request merged**